### DISCUSSION TITLE
Enable/disable action depending on other value

### DISCUSSION THREAD

**Discussion by PastCoder** [Zefau/ioBroker.jarvis#1040] [Q&A] [answered]
Hi,
probably there is an easy solution in Jarvis for the following need, but I did not find it yet:

- In normal mode one switch is managed automatically by a script in ioBroker
- But I can switch off the automation for a defined period of time
- During that time the user should be able to switch manually

How can I add or remove the trigger element depending on the mode? Or enable/disable the trigger element? Or hide/show it?

**Comment by mcuiobroker** [comment, marked as answer]
You can use Binding:
![image](https://user-images.githubusercontent.com/84818934/134817065-0ffa84a2-7a4c-4cce-b2cf-bfeb96cd6e45.png)

if the val is !=0 it is shown:
![image](https://user-images.githubusercontent.com/84818934/134817090-4d602506-8447-4414-b714-a8e6ceb5d68c.png)
if it is =0: you see nothing there
![image](https://user-images.githubusercontent.com/84818934/134817111-97e98c0e-7426-4360-ba1e-ba4ef99f61e0.png)

EDIT: 
`{"{0_userdata.0.testwert}=0":{"display":"none"}}`
It is the better way, because then the row disappear.

**Reply by PastCoder** [reply]
With jarvis-v3 or already in v2?

**Reply by Zefau** [reply]
v3
